source/MRMesh/MRConeObject.cpp

```diff
@@ -245,23 +255,43 @@ TEST( MRMesh, ConeApproximation )
 
     Cone3<float> resultCone;
     auto fit = Cone3Approximation<float>();
-    fit.solve( points, resultCone );
-    std::cout << "Cone apex: " << resultCone.center() << " direction:" << resultCone.direction() << " heigh:" << resultCone.height << " angle:" << resultCone.angle * 180.0f / PI_F << " (degree)" << std::endl;
+
+    Cone3ApproximationParams approxiamtorParams;
+    approxiamtorParams.coneFitterType = ConeFitterType::ApproximationPCM;
+
+    auto error = fit.solve( points, resultCone, approxiamtorParams );
+    std::cout << "Cone apex: " << resultCone.center() << " direction:" << resultCone.direction() << " heigh:" << resultCone.height << " angle:" << resultCone.angle * 180.0f / PI_F << " (degree)" << " error:" << error << std::endl;
```

Comment:
It will be printed anyway if next EXPECTs fail

Reply:
better use spdlog instad of std::cout

Reply:
Will change to spdlog

Reply:
just comment this code